### ISSUE TITLE
disable rhcd if registred via satellite/foreman ?

### ISSUE THREAD

**Issue by abdelhousni** [shadowman-lab/Ansible-ImageBuilder#4]
Hello @adworjan,

I encountered this loop-error :
```
...
Aug 28 08:52:44 pv-rh-imagebuilder rhcd[1254]: [rhcd] 2024/08/28 08:52:44 connection lost unexpectedly: use of closed network connection
Aug 28 08:52:44 pv-rh-imagebuilder rhcd[1254]: [rhcd] 2024/08/28 08:52:44 cannot publish message: connection timeout: 30s elapsed
...
```
And I found this article https://access.redhat.com/solutions/7019559
which recommends : 
> `rhcd `service is meant to be used only by systems registered directly to RHSM.

Is it to say that `rhcd` must be disable on systems registred to Satellite/Foreman ?

Thanks

Best regards,
Abdellatif

**Comment by adworjan** [comment]
Correct, the RHC client is specifically for systems directly managed by Red Hat Insights. Cloud Connector is used instead if your host is managed by Red Hat Satellite
